### DISCUSSION TITLE
Is it ideal to use bootstrap and scss together?

### DISCUSSION THREAD

**Discussion by gradaom** [twbs/bootstrap#36697] [Show & Tell]
While working on many projects , I see almost projects are using bootstrap and scss together . So is it ideal to use bootstrap and scss? Does Bootstrap contain scss , or these are each independent?

**Comment by florianlacreuse** [comment]
Bootstrap is written in Scss (Sass syntaxe) as you can see in the source code: https://github.com/twbs/bootstrap/tree/main/scss
For immediate and easy use, Bootstrap builds some compiled versions (css files): https://github.com/twbs/bootstrap/tree/main/dist/css
You can use Scss Bootstrap files to build you own custom css file based on Bootstrap.

https://getbootstrap.com/docs/5.2/customize/overview/
